content/docs/1.5.4/high-availability/recover-volume.md

```diff
@@ -27,7 +27,7 @@ The state of a volume can change to read-only when IO errors occur. IO errors ca
 - Network disconnection: Interrupted connection between the engine and replicas.
 - High disk latency: Significant delay in the transfer of data between a replica and the corresponding disk.
 
-Longhorn checks the state of the volume's global mount point every 10 seconds. When the volume's filesystem changes to read-only, Longhorn updates the condition to the volume's data engine. Longhorn then automatically attempts to remount the global mount point on the host to change the state back to read-write. Upon successful remounting, the workload pods continue functioning without disruption.
+Longhorn checks the state of the volume's global mount point every 10 seconds. When the volume's filesystem changes to read-only, Longhorn updates the condition to the volume's data engine. Longhorn then automatically attempts to remount the global mount point on the host to change the state back to read-write. Upon successful remounting, the workload pods continue functioning without disruption. However, if the mount point becomes write-protected and Longhorn fails to remount the mount point, you may still need to manually recreate the workload to force it reattach and remount the volume. 
```

Comment:
```suggestion
Longhorn checks the state of the volume's global mount point every 10 seconds. When the volume's filesystem changes to read-only, Longhorn updates the condition to the volume's data engine. Longhorn then automatically attempts to remount the global mount point on the host to change the state back to read-write. Upon successful remounting, the workload pods continue functioning without disruption. Longhorn also attempts to remount when the volume's filesystem changes to write-protected. If the remounting attempt fails, you may need to manually recreate the workload to force it reattach and remount the volume. 
```

Reply:
The sentence is too long so I separated the system action and user action.

Reply:
Hi @jillian-maroket 
I feel like the meaning has changed. 
Longhorn remount the mount point if it is now read-only.
However, the mount point can be read-only and also write protected. In this case, Longhorn won't be able to remount it to read-write.

That is, Longhorn only remount the mount point if it is read-only. The "write protected" is the case that Longhorn won't be remount the mount point.

Reply:
@ChanYiLin Thank you for clarifying. Let's use your original wording. I can't think of a way to separate the many concepts without altering the meaning.